homeassistant/components/media_player/plex.py

```diff
@@ -103,9 +105,14 @@ def setup_plexserver(host, token, hass, config, add_devices_callback):
     """Set up a plexserver based on host parameter."""
     import plexapi.server
     import plexapi.exceptions
+    configurator = get_component('configurator')
```

Comment:
Please move this line back to where it was originally. The configurator is not needed when setting up without the token.